CHANGES.txt

```diff
@@ -1,6 +1,12 @@
 CHANGES
 =======
 
+0.4.6 (2023-04-17)
```

Comment:
I don't think we can set the date until we do an actual release. I'd probably use "unrelated" or something like that.

Reply:
Changed to "undefined".